.github/workflows/signsbom.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: SignSBOM
+
+on:
+  pull_request:
+    branches: [master]
+    paths:
+      - ".github/workflows/signsbom.yml"
+      - "cyclonedx-lib/**"
+
+# Cancel existing runs if user makes another push.
+concurrency:
+  group: "${{ github.ref }}"
```

Comment:
This line is incorrect, which is why the SignSBOM does not always show in the CI checks, change it to this:
```
  group: ${{ github.workflow }}-${{ github.ref }}
```